### PR TITLE
added GHA to validate meta.yaml, transaction.yaml, registry.yaml

### PR DESCRIPTION
**Pull request by Deekshithrathod**
Tried to address issue #366 by adding GHA that validates these ( meta.yaml, transaction.yaml, registry.yaml) yaml files upon raising a Pull Request.

### CODE REVIEW

File: .github/workflows/openapi_validation.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Validate OpenAPI Schema
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (shenoyninad) [inline .github/workflows/openapi_validation.yml:6]: Can we have this for the draft branch too? Generally all changes are pushed to draft before they are moved to master.

Comment (Deekshithrathod) [reply .github/workflows/openapi_validation.yml:6]: Sure, that can be done. Will get it done, by tomorrow EOD.

**Comment by ravi-prakash-v** [comment]
@Deekshithrathod can you please submit your PR to the `draft` branch?

**Comment by Deekshithrathod** [comment]
@ravi-prakash-v do you want me to close this one & open new one on the draft branch?

**Comment by Deekshithrathod** [comment]
Sorry, @ravi-prakash-v @shenoyninad it took me this much time to make just a one-line change. Anywho, this is one.

**Comment by ravi-prakash-v** [comment]
Yes @Deekshithrathod I am closing this PR. Can you please create a new PR to draft?

**Comment by Deekshithrathod** [comment]
@emmayank done.

**Pull request closed without merging**